runtime-display/libdisplay-info/spec

```diff
@@ -0,0 +1,4 @@
+VER=0.1.1
+SRCS="https://gitlab.freedesktop.org/emersion/libdisplay-info/-/archive/$VER/libdisplay-info-$VER.tar.gz"
```

Comment:
Please try not to use automatically packaged tarballs, use Git tags instead.

Reply:
Fixed.